app/src/main/res/layout/item_list.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (C) 2017 Raul Hernandez Lopez @raulh82vlc
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<LinearLayout
```

Comment:
`ConstrainsLayout`? :D

Reply:
for a complex view with good animations for sure I'd go for it, for only a tuple into a simple adapter seems to be a little overhead to me (matter of personal preference of course). I'd use a _CoordinatorLayout'em_ all approach, at a more complex and nicer (visually speaking) side project, this is purely for trying to learn a little bit of functional really